### PR TITLE
Homework Javascript1 Week1

### PR DESCRIPTION
**Pull request by ItsNaklov**

### CODE REVIEW

File: javascript/javascript1/week1/homework.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+const yearFuture = 2070;
+
+const age = yearFuture - yearOfBirth;
+
+console.log("You will be " + age + " years old in " + yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 2012;
+const dogYearFuture = 2024;
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/homework.js:13]: The assignment descriptions assume dogYearFuture to be 2027 nd dogYearOfBirth 2010. Minor thing, and not as important as if the code actually works.

File: javascript/javascript1/week1/homework.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+const shouldShowResultInDogYears = true;
+const dogAge = dogYearFuture - dogYearOfBirth;
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/homework.js:15]: This variable name is a bit confusing, especially when you have another called `dogYears`. Could this one be called `humanYears`, `regularYears` or something else similar?

File: javascript/javascript1/week1/homework.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  height: 10,
+  gardenSize: 100,
+};
+
+const juliaHouse = {
+  width: 5,
+  depth: 11,
+  height: 8,
+  gardenSize: 70,
+};
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/homework.js:43]: Those objects are used just ones. You could have simplified your code by directly using the values to call the functions further down in const `peterHousePrice` and `juliaHousePrice`

File: javascript/javascript1/week1/homework.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+  console.log(`Your dog will be ${dogYears} dog years old in ${dogYearFuture}`);
+} else {
+  console.log(`Your dog will be ${dogAge} human years old in ${dogYearFuture}`);
+}
+
+//Housey pricey (A house price estimator)
+function calculateHousePrice(width, depth, height, gardenSize) {
+  const volumeInMeters = width * depth * height;
+  const housePrice = volumeInMeters * 2.5 * 1000 + gardenSize * 300;
+  return housePrice;
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/homework.js:28]: You return the result of the formula for the house price, but no message to indicate if they have paid too much or too little depending on that price given in the assignment. You need to compare `housePrice` with 2.500.000 for Peter's house and log "Too high" or "Too low" depending on the result - same for Julia.

File: javascript/javascript1/week1/homework.js [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+  "Cybersecurity",
+  "Dataanalytics",
+  "Systems",
+  "Services",
+  "World",
+  "Future",
+];
+
+const randomFirstWordIndex = Math.floor(Math.random() *10);
+const randomSecondWordIndex = Math.floor(Math.random() *10);
```
Comment (jason-vasilev) [inline javascript/javascript1/week1/homework.js:89]: Perfect! This will give you the biggest mix of name combinations.

**Review by jason-vasilev — COMMENTED** [review]
I'm glad to see you've read about functions, used template literals, and prettier. Yet, I'd like you to reread the assignments as there are a few missing things. Remember to commit more often and add a more descriptive PR description.

**Comment by ItsNaklov** [comment]
Hello, 
I fixed the issues according review comments. I hope everything is correct.
BR

**Pull request merged**